### PR TITLE
[CI] use ocis repo for glauth

### PR DESCRIPTION
**Pull request by dpakach**
## Description

use glauth from the monorepo instead of ocis-glauth repo(old version) in CI.
We do not run test using openIdConnect in CI currently, but all the necessary setup for openIdConnect is still in .drone.star.
This is because the setup described in https://owncloud.dev/ocis/deployment/bridge/ doesn't works currently.
This Pr just updates the repo from which glauth for when we again run tests using openIdConnect In CI.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/4888

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15467/

<p>


### webUIUpload-upload-feature-130.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15467/

<p>


### webUIMoveFilesFolders-moveFiles-feature-105.png
  


</p></details>

File: .drone.star [hunk 1]
```diff
@@ -1593,16 +1593,10 @@ def buildGlauth():
...
-            # using ocis-glauth repo because latest glauth doesn't supports the bridge setup
-            "mkdir -p /srv/app/src",
-            "cd /srv/app/src",
-            "mkdir -p github.com/owncloud/",
-            "cd github.com/owncloud/",
-            "git clone http://github.com/owncloud/ocis-glauth",
-            "cd ocis-glauth",
-            "git checkout 44e252306af2dedd72ad00567bbfe9c03322ab20",
+            "cd /srv/app/src/github.com/owncloud/ocis",
+            "cd glauth",
```
Comment (kiranparajuli589) [inline .drone.star:1597]:
```suggestion
            "cd /srv/app/src/github.com/owncloud/ocis/glauth",
```

**Comment by phil-davis** [comment]
Only 1 scenario failed: https://drone.owncloud.com/owncloud/web/15485/64/9
```
runsh: Total unexpected failed scenarios throughout the test run:
webUIRenameFiles/renameFiles.feature:130
```

Maybe that is one of the intermittent failures. I will restart CI.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-107.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-50.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-68.png
  

### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases-feature-86.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-42.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-58.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-74.png
  

### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersBlacklisted</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-26.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-44.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-62.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-80.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-92.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-104.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-114.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-124.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-22.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-42.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-58.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-74.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-89.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-100.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-121.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-139.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-155.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-165.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-17.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-58.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-80.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUIMarkdownEditor-markdownFile-feature-12.png
  

### webUIMarkdownEditor-markdownFile-feature-19.png
  

### webUIMarkdownEditor-markdownFile-feature-26.png
  

### webUIMarkdownEditor-markdownFile-feature-33.png
  

### webUIMarkdownEditor-markdownFile-feature-39.png
  

### webUIMarkdownEditor-markdownFile-feature-44.png
  

### webUIMarkdownEditor-markdownFile-feature-49.png
  

### webUIMarkdownEditor-markdownFile-feature-62.png
  

### webUIMarkdownEditor-markdownFile-feature-63.png
  

### webUIMarkdownEditor-markdownFile-feature-71.png
  

### webUIMarkdownEditor-markdownFile-feature-72.png
  

### webUIMarkdownEditor-markdownFile-feature-73.png
  

### webUIMarkdownEditor-markdownFile-feature-74.png
  

### webUIMarkdownEditor-markdownFile-feature-75.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersExpireToRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-100.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-126.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-14.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-171.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-172.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-199.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-200.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-203.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-21.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-220.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-232.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-30.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-48.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-76.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-77.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-96.png
  

### webUISharingInternalUsersExpireToRoot-shareWithUsersExpiringShares-feature-97.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersToRoot-shareWithUsers-feature-103.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-119.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-129.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-142.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-161.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-183.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-197.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-212.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-255.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-256.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-257.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-260.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-267.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-275.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-52.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-53.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-54.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-57.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-74.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalGroupsToRoot-shareWithGroups-feature-128.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-167.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-179.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-197.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-209.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-21.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-222.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-234.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-245.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-265.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-293.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-294.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-315.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-316.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-69.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-70.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-71.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-74.png
  

### webUISharingInternalGroupsToRoot-shareWithGroups-feature-95.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersExpire</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-104.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-105.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-108.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-134.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-16.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-179.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-180.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-207.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-208.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-211.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-228.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-24.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-240.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-34.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-54.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-84.png
  

### webUISharingInternalUsersExpire-shareWithUsersExpiringShares-feature-85.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-100.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-109.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-31.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-32.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-33.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-34.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-37.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-44.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-54.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-74.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-75.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-78.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-86.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-93.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-11.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-25.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-40.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-54.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-61.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-14.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-26.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-40.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-52.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-61.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-70.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-79.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-88.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-96.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15492/

<p>


### webUISharingInternalUsers-shareWithUsers-feature-112.png
  

### webUISharingInternalUsers-shareWithUsers-feature-130.png
  

### webUISharingInternalUsers-shareWithUsers-feature-141.png
  

### webUISharingInternalUsers-shareWithUsers-feature-162.png
  

### webUISharingInternalUsers-shareWithUsers-feature-184.png
  

### webUISharingInternalUsers-shareWithUsers-feature-199.png
  

### webUISharingInternalUsers-shareWithUsers-feature-233.png
  

### webUISharingInternalUsers-shareWithUsers-feature-279.png
  

### webUISharingInternalUsers-shareWithUsers-feature-280.png
  

### webUISharingInternalUsers-shareWithUsers-feature-281.png
  

### webUISharingInternalUsers-shareWithUsers-feature-321.png
  

### webUISharingInternalUsers-shareWithUsers-feature-330.png
  

### webUISharingInternalUsers-shareWithUsers-feature-341.png
  

### webUISharingInternalUsers-shareWithUsers-feature-350.png
  

### webUISharingInternalUsers-shareWithUsers-feature-57.png
  

### webUISharingInternalUsers-shareWithUsers-feature-58.png
  

### webUISharingInternalUsers-shareWithUsers-feature-59.png
  

### webUISharingInternalUsers-shareWithUsers-feature-82.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-35.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-36.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-37.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-38.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-41.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-51.png
  

### webUISharingInternalUsersCollaborator-shareWithUsers-feature-65.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-119.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-136.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-155.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-156.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-159.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-168.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-17.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-176.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-186.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-199.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-31.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-48.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-60.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-70.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-81.png
  

### webUISharingInternalUsersShareWithPage-shareWithUsers-feature-92.png
  


</p></details>

**Review by phil-davis — APPROVED** [review]
LGTM. @dpakach were the "random" test failures in the first CI runs just "random"? Or do you think that the "new" glauth might be more flaky?

**Pull request merged**